Core/include/Acts/Geometry/GeometryObjectSorter.hpp

```diff
@@ -8,10 +8,8 @@
 
 #pragma once
 
-// clang-format off
 // Workaround for building on clang+libstdc++. Must always be first
 #include "Acts/Utilities/detail/ReferenceWrapperAnyCompat.hpp"
-// clang-format on
```

Comment:
I put these there so this never gets reordered by clang-format. Are you sure it won't reorder?

Reply:
I can revert this. just noticed that we have a couple of occasions without that format guard